Update newmeta_spec.yaml

dosumis opened the pull request:
Moved spec from common_fields to newmeta_spec

dosumis committed:
Update newmeta_spec.yaml

Moved spec from common_fields to newmeta_spec

Clare72 reviewed (approved):
Looks fine, perhaps also add a link to the example newmeta_Jovanovic2016_191106.yaml file from the wiki

dosumis committed:
Update newmeta_Jovanovic2016_191106.yaml

dosumis committed:
Update newmeta_Jovanovic2016_191106.yaml

list -> string.  Curator field needs some work!

dosumis commented:
Added to wiki and changed example file to use an orcid.  Actually - I haven't implemented processing of this field yet for new_meta so going nowhere right now.  Maybe worth discussing  if we should use actual orcid or whether we allow initials and implement a lookup.
